app/gate/api.js

```diff
@@ -2,9 +2,9 @@ import axios from 'axios';
 import isEmpty from 'lodash/isEmpty';
 
 import tokenHelper from 'helpers/token';
-import {apiUrl} from 'helpers/config';
+import {API_URL} from 'react-native-dotenv';
```

Comment:
Why the new linter configuration doesn't show an error? line 5 should write like below:

    import { API_URL } from 'react-native-dotenv';

Does the linter omit the spaces for a curly brace?

Reply:
yes, this is done by linter